src/styles/elements/_tags.scss

```diff
@@ -0,0 +1,18 @@
+.tag {
```

Comment:
Consider creating a simple component e.g. `BaseTag` instead of creating global styles - it is much easier to maintain it or make some changes to it in the future 🤞 

Example similar component from Vuetify:
https://vuetifyjs.com/en/components/chips

See how many different options may appear over time - believe me it is much safer to make such changes in the component than to change global styles (where e.g. html structure is not enforced)